src/tests/multihost/ad/test_ad_misc.py

```diff
@@ -170,3 +170,60 @@ def test_0001_provider_config_cross_interference(
         assert "[sdap_cli_auth_step] (0x1000): Invalid authtoken type" \
             not in log_str, "The configuration interferes."
         assert "Going offline" in log_str
+
+    @staticmethod
+    def test_0002_improved_use_negative_sid_for_sid_lookup(
+            multihost, adjoin, backupsssdconf):
+        """
+        :title: IDM-SSSD-TC: Multiple provider configuration interferes
+         with each other
+        :bugzilla:
+          https://bugzilla.redhat.com/show_bug.cgi?id=1766490
+        :id: fc07a12b-58de-49f4-93be-4fa2b8cdc6ee
+        :setup:
+          1. Configure sssd with AD.
+          2. Enable debug_level = 9 in the domain section
+          3. Transport a python program, to run a lookup using SID, on client
+        :steps:
+          1. Twice run lookup using sid of non-existing ADuser
+        :expectedresults:
+          1. domain log file should show that the SID of non-existing does
+             not exist in negative cache
+        """
+        tools = sssdTools(multihost.client[0])
+        client = sssdTools(multihost.client[0])
+        domain_name = tools.get_domain_section_name()
+        multihost.client[0].service_sssd('stop')
+        domain_section = f'domain/{domain_name}'
+        sssd_params = {
+            'debug_level': '9',
+            'use_fully_qualified_names': 'True'
+        }
+        client.sssd_conf(domain_section, sssd_params, action='add')
+        tools.clear_sssd_cache()
+        multihost.client[0].run_command('dnf install python3-libsss_nss_idmap -y', raiseonerr=True)
```

Comment:
If I look at `ipa/test_adtrust.py` I have impression that this package is installed by default. Am I wrong?

Reply:
I'm not sure about test_adtrust.py, however this test fails and complains about missing python3-libsss_nss_idmap.py. It seems installation of python3-libsss_nss_idmap is required for this test.